.circleci/config.yml

```diff
@@ -125,6 +125,20 @@ commands:
               echo "detected non-webui-only changes: $DIFF_DIRS"
             fi
 
+  skip-if-not-shared:
+    steps:
+      - run: git fetch upstream
+      - run:
+          name: check for shared changes (must be run after skip-if-docs-only)
```

Comment:
nit: is this "must be run after.." needed ? 
maybe could be a comment 🤔 the name shows up in the ui

Reply:
ok i moved to comment.

the situation was they both do `git fetch upstream` and if you try to add an existing remote it errors. 

so just made it where you add the remote in one and then reuse it in the other. feels gross but w/e

Reply:
could clean it up a bit, but maybe next time around lol.

Reply:
it turns out putting it into `test-intg` made it no longer have access to remote.

@hamidzr do you think it's okay to do 
```
git remote add upstream https://github.com/determined-ai/determined 2>/dev/null
git fetch upstream
```

that way adding error for existing remote is supressed (but fetch fail will still cause error)

and we can get rid of dependency between `skip-webui` and `skip-docs`, making things simpler